app/script/util/util.js

```diff
@@ -148,6 +148,19 @@ z.util.getFileExtension = filename => {
   return filename.substr(filename.lastIndexOf('.') + 1);
 };
 
+/**
+ * Trim end of a string.
+ * @param {string} text - text to trim
+ * @returns {string} Trimmed string
+ */
+z.util.trimEnd = text => {
+  if (_.isString(text)) {
```

Comment:
I would not put safeguards here. We probably want to know if we call this with an unexpected input

Reply:
Also how about do feature detection here.

```
return text.trimEnd ? text.trimEnd() : text.replace(...)
```